### ISSUE TITLE
Uncaught Error: end pattern with unmatched parenthesis

### ISSUE THREAD

**Issue by dgrothman** [atom/atom#5565]
[Enter steps to reproduce below:]

1. ...Open custom wordpress plugin file in development
2. ...Change Document Mode in bottom right from PHP to wordpress

**Atom Version**: 0.179.0
**System**: Microsoft Windows 8.1
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: end pattern with unmatched parenthesis

```
At C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\scanner.js:31

Error: end pattern with unmatched parenthesis
  at Error (native)
  at Scanner.module.exports.Scanner.createScanner (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\scanner.js:31:24)
  at Scanner.module.exports.Scanner.getScanner (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\scanner.js:37:31)
  at Scanner.module.exports.Scanner.findNextMatch (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\scanner.js:56:22)
  at Rule.module.exports.Rule.findNextMatch (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\rule.js:101:28)
  at Rule.module.exports.Rule.getNextTokens (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\rule.js:143:21)
  at Grammar.module.exports.Grammar.tokenizeLine (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\first-mate\lib\grammar.js:125:39)
  at TokenizedBuffer.module.exports.TokenizedBuffer.buildTokenizedLineForRow (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\src\tokenized-buffer.js:482:28)
  at TokenizedBuffer.module.exports.TokenizedBuffer.tokenizeNextChunk (C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\src\tokenized-buffer.js:265:43)
  at C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\src\tokenized-buffer.js:247:26
  at C:\Users\Daniel\AppData\Local\atom\app-0.179.0\resources\app\node_modules\roaster\node_modules\underscore\underscore.js:666:47

```

### Commands

```
     -3:35.9 command-palette:toggle (atom-text-editor.editor)
     -3:30.3 core:backspace (atom-text-editor.editor.mini)
     -2:48.9 project-find:show (atom-text-editor.editor)
     -2:44.4 editor:consolidate-selections (atom-text-editor.editor.mini)
     -2:44.4 core:cancel (atom-text-editor.editor.mini)
     -2:42.6 command-palette:toggle (atom-text-editor.editor)
     -2:18.3 snippets:available (atom-text-editor.editor)
     -1:56.4 command-palette:toggle (atom-text-editor.editor)
     -1:38.5 timecop:view (atom-text-editor.editor)
     -0:40.1 editor:consolidate-selections (atom-text-editor.editor)
     -0:40.1 core:cancel (atom-text-editor.editor)
 10x -0:39.2 core:backspace (atom-text-editor.editor)
     -0:35.7 grammar-selector:show (atom-text-editor.editor)
     -0:32.3 core:confirm (atom-text-editor.editor.mini)
     -0:06.6 grammar-selector:show (atom-text-editor.editor)
     -0:03.7 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.2.0
wordpress-api, v1.1.0

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
This is caused by the same root problem as #5472 which will be fixed in atom/notifications#39.
In the meantime, please open this issue on the [wordpress-api](https://github.com/azizur/atom-wordpress-api/issues) package.

**Comment by DavidLGoldberg** [comment]
+1
